src/sentry/static/sentry/app/components/panels/panelAlert.jsx

```diff
@@ -13,15 +13,15 @@ const DEFAULT_ICONS = {
 
 // Margin bottom should probably be a different prop
 const PanelAlert = styled(({type, icon, ...props}) => (
-  <Alert {...props} icon={icon || DEFAULT_ICONS[type]} type={type} system={true} />
+  <Alert {...props} icon={icon || DEFAULT_ICONS[type]} type={type} system />
 ))`
   margin: 0 0 1px 0;
   border-radius: 0;
 `;
 
 PanelAlert.propTypes = {
+  ...Alert.propTypes,
   type: PropTypes.oneOf(['info', 'warning', 'success', 'error', 'default']),
```

Comment:
Does default need to be muted here as well?

Reply:
yep, good catch